### ISSUE TITLE
Search form on home page is out of view

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2098]
1. Go to home page
2. Click on search.

The search area disappears outside of the page.

![capture d ecran 2018-02-16 a 12 25 20 2](https://user-images.githubusercontent.com/505230/36292397-f67ad3d6-1314-11e8-8e04-c8bb28cad9d5.png)

**Comment by softvision-oana-arbuzov** [comment]
This issue is reproducible also on mobile app.
![screenshot-1519027872140](https://user-images.githubusercontent.com/12184325/36367859-418b3fd8-155d-11e8-81ee-4004a91ded62.jpg)

Affected area:
```html
<form id="search-bar" class="grid" method="get" action="/issues">
	<input class="form-field text-field" name="q" placeholder="Search issues by keyword">
		<button class="nav-button js-SearchBarClose" aria-hidden="true" type="button">
			<svg class="icon nav-icon" viewBox="0 0 30 30" role="presentation">
				<title>Close search</title>
				<use href="#svg-x">Close search</use>
			</svg>
    	</button>
</form>
```

Modifying the `position` from `absolute` to `relative`, the "Search" bar becomes visible.
```css
#search-bar {
    position: absolute;
    top: -27px;
    z-index: 1200;
    max-width: 900px;
    min-width: 270px;
    width: 100%;
    -webkit-transform: rotateY(-90deg);
    transform: rotateY(-90deg);
    -o-transition: all .2s ease 0s;
    transition: all .2s ease 0s;
    margin-left: auto;
    margin-right: auto;
    left: 0;
    right: 0;
}
```

![screenshot-1519028147789](https://user-images.githubusercontent.com/12184325/36367977-d20450a4-155d-11e8-949b-9e2dafe3b7d2.jpg)

**Tested with:**
Browser / Version: Firefox Nightly Mobile 60.0a1 (2018-02-18)
Operating System: Google Pixel (Android 8.1.0) - Resolution 1080 x 1920 pixels (~441 ppi pixel density)

**Comment by zoepage** [comment]
This module needs fixing in responsive mode. I'll assign myself.

**Comment by zoepage** [comment]
merged in #2098
